imap/vcard_support.c

```diff
@@ -126,6 +126,21 @@ EXPORTED struct vparse_card *record_to_vcard(struct mailbox *mailbox,
     return vcard;
 }
 
+static const struct image_signature {
+    const uint8_t magic[8];
+    size_t magiclen;
+    const char *mediatype;
+} image_signatures[] = {
```

Comment:
Wikipedia lists a couple of additional magic bytes for the respective types:

# JPG
```
FF D8 FF DB
FF D8 FF E0 00 10 4A 46
49 46 00 01
FF D8 FF EE
FF D8 FF E1 ?? ?? 45 78
69 66 00 00
```

where `??` is the data size.

# Google WebP format (`image/webp`)
```
52 49 46 46 ?? ?? ?? ??
57 45 42 50
```

where `??` is the file size.